styles/styles.css

```diff
@@ -0,0 +1,248 @@
+  background-size: cover;
+  background-color: #cccccc;
+} */
+
+li {
+  list-style: none;
+}
+
+.grid-container {
+  display: grid;
+  height: 100%;
+  /* grid-template-rows: 1fr 4fr 2fr 2fr 2fr 4fr 3fr 2fr; */
+}
+
+.top-nav {
+  display: grid;
+  grid-template-columns: 70% 15% 15%;
+  grid-template-rows: 3rem;
+  list-style-type: none;
+}
+
+#title {
+  font-family: 'Quicksand', sans-serif;
+  grid-column-start: 1;
+}
+
+#try-now {
+  font-family: 'Quicksand', sans-serif;
+  grid-column-start: 2;
+  text-align: center;
+}
+
+#about {
+  font-family: 'Quicksand', sans-serif;
+  grid-column-start: 3;
+  text-align: center;
+}
+
+h1 {
+  font-family: 'PT Sans', sans-serif;
+  font-weight: bold;
+  font-size: 5rem;
+}
+
+.home-container sub {
+  height: 2rem;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 2rem;
+}
+
+#call-action {
+  font-family: 'PT Sans', sans-serif;
+  font-size: 1rem;
+  height: 1.5rem;
+  color: black;
+  border: 1px solid grey;
+  border-radius: 8px;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 1.2rem;
+  background-color: white;
+}
+
+.home-container {
+  background-image: "trianglify.png"; 
+  background: linear-gradient(white, aqua);
+  height: 48rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 3em;
+  border-bottom: solid thin gray;
+}
+
+.quote-grid {
+  /* margin-left: 8rem;
+  margin-right: 8rem; */
+  display: grid;
+  height: 15rem;
+  grid-template-rows: 8rem;
+  grid-template-columns: 20% 60% 20%;
+  border-bottom: solid thin gray;
+}
+
+.quote {
+  grid-row: 1;
+  grid-column: 2;
+  font-style: italic;
+  font-size: 1.4rem;
+  text-align: left;
+  font-family: 'Quicksand', sans-serif;
+  padding-top: 2rem;
+}
+
+.author {
+  grid-row: 2;
+  grid-column: 2;
+  font-size: 1.2em;
+  text-align: right;
+  font-family: 'Quicksand', sans-serif;
+}
+
+.why-grid {
+  background-color: aqua;
+  justify-content: center;
+  width: 100%;
+  font-family: 'Quicksand', sans-serif;
+  height: 20rem;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  /* background-color: blueviolet; */
+  padding-bottom: 4rem;
+  border-bottom: solid thin gray;
+  /* grid-template-columns: 1 1 1; */
+}
+
+#why-element {
+  grid-row: 1;
+  text-align: center;
+  font-weight: bold;
+  font-size: 2rem;
+}
+
+.images-container {
+  grid-row: 2;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 100%;
+  /* grid-gap: 4rem; */
+}
+
+.image-item {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  width: 300px;
+  /* grid-template-rows: 1fr 1fr; */
+}
+
+.image-item img {
+  grid-column: 1;
+  grid-row: 1 / span 2
+}
+
+.image-item strong {
+  grid-column: 2;
+  grid-row: 1;
+  font-size: 1.2rem;
+}
+
+.image-item p {
+  padding-top: 0.5rem;
+  grid-column: 2;
+  grid-row-start: 1;
+}
+
+.team-members {
```

Comment:
I love your use of a grid in this area!! I struggled so much with this section, and yours looks so clean and perfectly spaced. Great job!